### DISCUSSION TITLE
PVinverter is shown as "Fronius" in VRM

### DISCUSSION THREAD

**Discussion by americanium** [freakent/dbus-mqtt-devices#103] [General]
Hello!

Not really a problem of this code I think, but when implementing my sungrow inverter as "pvinverter" it is shown as "Fronius" pvinverter.
I have another fronius in use, but i wan't to seperate them in two single inverters within VRM.

Thank you!
Alex

**Comment by freakent** [comment]
Without any code examples or screenshots it's impossible to give you any advice.

**Comment by americanium** [comment]
Hi!
Sorry for that.

This is my current dashboard
![image](https://github.com/freakent/dbus-mqtt-devices/assets/96286886/78d30b8a-ed3d-4b3d-9749-37502bf4c8c2)

You see the "Fronius" which is my real fronius added by VRM automatically via (modbus?)

And here my current configuration in node red for testing adding another PV inverter.

![image](https://github.com/freakent/dbus-mqtt-devices/assets/96286886/343223b2-00eb-4a03-b694-de476e5b050a)

The pvinverter is listed within vrm device overview with extra VRM instance, but the values are added to the already existing fronius inverter.

![image](https://github.com/freakent/dbus-mqtt-devices/assets/96286886/cfec04a8-cb58-4572-8420-c95e314a733f)

Do you need any specific part of code or screenshot ?

Thanks!

**Comment by freakent** [comment]
The GX UI is very limited. I suspect it will not show multiple PVInverters. I have no experience with PVInverters but I do have 3 mppts in my installation, they are represented by one Mppt icon and the data aggregated. The Icon is just a generic Mppt icon, not a picture of my actual MPPT models. 

![image](https://github.com/freakent/dbus-mqtt-devices/assets/124850/365dc822-4893-406f-9b76-5c624204f9e3)

>You see the "Fronius" which is my real fronius added by VRM automatically via (modbus?)

I'm not sure what you mean by this. Nothing is added by VRM, VRM is the portal in Victron's cloud.

**Comment by americanium** [comment]
Sorry, I didn't mean VRM but the VenusUI.

I will cheeck victron forum for a solution, here you see pv inverters without "fronius" symbol:
https://community.victronenergy.com/questions/3848/how-to-add-pv-meter-on-ac-out1-multigrid-venus.html
